.github/workflows/release_workflow.yml

```diff
@@ -21,6 +21,10 @@ on:
         default: false
 jobs:
   security:
+    permissions:
+      # needed to write security info to repository
+      security-events: write
```

Comment:
If a permission is not explicitly listed what is default permission? What is defined as the default workflow permission in repo settings?

Reply:
If no permissions, are listed - the default permission is `read`, but the docs state that any `permissions` entries will set other permissions defaults to `none` unless otherwise specified.

Something else I discovered in testing is that parent jobs need to define the same permissions as requested by any reusable workflows they reference